varnish.vcl

```diff
@@ -11,6 +11,14 @@ acl purge {
 
 sub vcl_recv {
 
+    #Ref #142770
+    if (req.url ~ "api//SITE/ims") {
+        if (! req.http.Accept ~ "json") {
+           set req.url = regsub(req.url, "/VirtualHostBase/https/staging.eea.europa.eu:443/www/VirtualHostRoot/_vh_api//SITE", "");
```

Comment:
@anton16 I don't think this line will work on production as it contains staging within.

Reply:
Also https://staging.eea.europa.eu/ims doesn't work anymore.